content.js

```diff
@@ -1,11 +1,11 @@
 /* global chrome, PHRASES_TO_HIDE */
 
 (function () {
-  const itemClass = 'TimelineItem';
-  const itemHiddenClass = 'TimelineItem-Hidden';
-  const toggleButtonID = 'tidy-timeline-button';
+  const itemClass = "TimelineItem";
```

Comment:
Shall we revert these formatting changes and add prettier config?

Reply:
Sure, happy with that? Do you have the prettier config you used to format the project?

Reply:
Agree; single quotes are a defacto JS standard, and used in other M&S repos as well.

Reply:
I have added prettier and editor config